contentctl/actions/detection_testing/views/DetectionTestingView.py

```diff
@@ -93,16 +94,27 @@ def getSummaryObject(
             )
 
             # Aggregate detection pass/fail metrics
-            if summary["success"] is True:
-                total_pass += 1
-            else:
+            if summary["success"] is False:
                 total_fail += 1
+            else:
+                #Test is marked as a success, but we need to determine if there were skipped unit tests
+                #SKIPPED tests still show a success in this field, but we want to count them differently
+                pass_increment = 1
+                for test in summary.get("tests"):
+                    if test.get("test_type") == "unit" and test.get("status") == "skip":
```

Comment:
I think, and I don't need this to be blocking, but I'd like to make detection state resolution (e.g. is it considered a skip/fail/error/pass) a property of the detection itself

E.g. define a property function `status` which iterates over tests
- Pass: If at least one test passes and the rest pass/skip
- Skip: If all tests skip
- Fail: If at least one test fails and the rest pass/skip/fail
- Error: If at least one detection errors (regardless of other test states)

Reply:
💯 
I think we should make this a pydantic computed_property.  On top of that, I would add an additional option to this set of values (which is the default value) along the lines of NOT_YET_TESTED.
This way, when we are rendering the current state of all tests, like in a user interface, we can easily differentiate /sort all sorts of different tests.